html/changelogs/geeves - manifesto.yml

```diff
@@ -0,0 +1,6 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - rscadd: "Stationbounds have been added to the Crew Manifest in a Sub-manifest catagory. They are not crew."
```

Comment:
```suggestion
  - rscadd: "Stationbounds have been added to the Crew Manifest in a Station Equipment catagory. They are not crew. It will also not appear on PDA's (yet)."
```